server/routerlicious/packages/lambdas/src/utils/messageGenerator.ts

```diff
@@ -48,3 +48,35 @@ export const createRoomLeaveMessage = (clientId: string): ISignalMessage => ({
 		content: clientId,
 	}),
 });
+
+export const IBroadcastSignalEventPayloadType = {
+	/**
+	 * Indicates that the data associated with an edit is or must be a `boolean`.
+	 */
+	RuntimeMessage: "RuntimeMessage",
+} as const;
+
+export const IRuntimeSignalMessageBody = {
+	/**
+	 * Indicates that the data associated with an edit is or must be a `boolean`.
+	 */
+	content: "RuntimeMessage",
+} as const;
+
+export interface IRuntimeSignalMessageBody {
+	/**
+	 * Mirrors ISignalEnvelope from runtime defitions. Contains the contents of the envelope
+	 */
```

Comment:
I think I'd move this to be the docs of the interface as a whole, and does this sound accurate?

`Mirrors ISignalEnvelope from runtime definitions, for signals that come from an external caller (not sent by a client (so no 'clientSignalSequenceNumber') and are always addressed to the Container (so no 'address'`).

And what do you think about calling it `IRuntimeSignalEnvelope`, to further clarify the mirroring?

Reply:
Yes that's a much better description.